npm/private/npm_translate_lock_helpers.bzl

```diff
@@ -278,7 +278,6 @@ def _get_npm_imports(importers, packages, patched_dependencies, only_built_depen
         optional_deps = package_info.get("optional_dependencies")
         dev = package_info.get("dev")
         optional = package_info.get("optional")
-        pnpm_patched = package_info.get("patched")
```

Comment:
This property came directly from the lockfile and it no longer exists in v9. However the `patched_dependencies` dict has already been computed before this method is called so we can just check if the package exists there. Assuming `patched_dependencies` was constructed correctly, but we probably make that assumption already? 😅